### PR TITLE
Adding Unit tests to metric_store.go

### PR DESCRIPTION
**Pull request by muaazsaleem**
# One-line summary

> Issue : https://github.com/zalando-incubator/kube-metrics-adapter/issues/6

## Description
Adding unit tests for `metric_store.go`

## Types of Changes
_What types of changes does your code introduce? Keep the ones that apply:_

- Refactor/improvements

## Tasks
_List of tasks you will do to complete the PR_
  - https://github.com/zalando-incubator/kube-metrics-adapter/issues/6

## Review
_List of tasks the reviewer must do to review the PR_
- [ ] Tests

### CODE REVIEW

**Comment by muaazsaleem** [comment]
Tests are failing because `metric.TTL` isn't configurable yet.

File: pkg/provider/metric_store.go [hunk 1]
```diff
@@ -77,7 +81,7 @@ func (s *MetricStore) insertCustomMetric(value custom_metrics.MetricValue, label
 	metric := customMetricsStoredMetric{
 		Value:  value,
 		Labels: labels,
-		TTL:    time.Now().UTC().Add(15 * time.Minute), // TODO: make TTL configurable
+		TTL:    metricsTTL(), // TODO: make metricsTTL configurable
```
Comment (mikkeloscar) [inline pkg/provider/metric_store.go]: I would rather we pass this as an argument to `NewMetricStore`

**Comment by mikkeloscar** [comment]
:+1:

**Comment by muaazsaleem** [comment]
👍

**Pull request merged**